Package.swift

```diff
@@ -23,7 +23,7 @@ let package = Package(
 
     // TODO: Add Linux / Windows support
     platforms: [
-        .macOS(.v10_15),
```

Comment:
Is there a valid reason for. doing this? We can't drop support for an OS without a good reason.